aca_preview/preview.py

```diff
@@ -336,3 +340,66 @@ def check_bright_guide_for_ers(self, n_bright_req=3, bright_lim=9.0):
+        is_OR = abs(obsid) < 38000
+        if not is_OR and len(self.guides) < n_required:
+            self.add_message('critical', f'ER guides stars: only {len(self.guides)} stars')
+
+    def check_guide_pos_errs(self):
+        """
+        Warn on stars with larger POS_ERR (warning at 1" critical at 2")
+        """
+        for star in self.guides:
+            agasc_id = star['id']
+            # POS_ERR is in milliarcsecs in the table
+            pos_err = star['POS_ERR'] * 0.001
+            for limit, category in ((2.0, 'critical'),
+                                    (1.0, 'warning')):
+                if pos_err > limit:
+                    self.add_message(category, f'Guide star {agasc_id} has POS_ERR {pos_err:.2f}, limit {limit} arcsec')
+                    break
+
+    def check_imposters(self):
+        """
+        Warn on stars with larger imposter centroid offsets
+        """
+
+        # Borrow the imposter offset method from starcheck
+        def imposter_offset(cand_mag, imposter_mag):
+                """
+                For a given candidate star and the pseudomagnitude of the brightest 2x2 imposter
+                calculate the max offset of the imposter counts are at the edge of the 6x6
+                (as if they were in one pixel).  This is somewhat the inverse of proseco.get_pixmag_for_offset
+                """
+                cand_counts = mag_to_count_rate(cand_mag)
+                spoil_counts = mag_to_count_rate(imposter_mag)
+                return spoil_counts * 3 * 5 / (spoil_counts + cand_counts)
+
+        for star in self.guides:
+            agasc_id = star['id']
+            imp_offset = imposter_offset(star['mag'], star['imp_mag'])
+            for limit, category in ((4.0, 'critical'),
+                                    (2.5, 'warning')):
+                if imp_offset > limit:
+                    self.add_message(category, f'Guide star {agasc_id} imposter offset {imp_offset:.1f}, limit {limit} arcsec')
+                    break
+
+    def check_too_bright_guide(self):
+        """
+        Warn on guide stars that may be too bright.
+        Set to warn with critical warn if MAG_ACA_ERR used in selection is less than 0.1 or if within 3 * mag_err of the hard 5.8 limit.
+        Set to warn with orange warn if just brighter than 6.1 (should be double-checked in context of other candidates).
+        """
+        for star in self.guides:
+            agasc_id = star['id']
+            if star['mag'] - (3 * star['mag_err']) < 5.8:
+                self.add_message('critical', f'Guide star {agasc_id} within 3*mag_err of 5.8')
+            elif (star['mag'] < 6.1) and (star['MAG_ACA_ERR'] * .01 < 0.1):
+                self.add_message('critical', f'Guide star {agasc_id} < 6.1 with small MAG_ACA_ERR.  Double check selection.')
+            elif star['mag'] < 6.1:
+                self.add_message('warning', f'Guide star {agasc_id} < 6.1. Double check selection.')
```

Comment:
I suspect this is going to be adding noise, but for now is fine while we get experience and fine-tune proseco.